CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Planner: Allow for a final dive segment at the surface
```

Comment:
I'm being picky, but is this a "dive" segment or is this a "deco" segment?

Reply:
Will do. Expect an update tonight (old world time).